api/swagger.yaml

```diff
@@ -5615,6 +5615,26 @@ definitions:
         items:
           type: "string"
         example: ["--debug", "--systemd-cgroup=false"]
+      status:
+        description: |
+          Information specific to the runtime.
+
+          <p><br /></p>
+
+          > **Note**: The information returned in this field, including the
+          > formatting of values and labels, should not be considered stable,
+          > unless explicitly documented here.
+          > Those information may change without notice.
+        type: "object"
+        x-nullable: true
+        properties:
+          features:
+            description: |
+              [features structure](https://github.com/opencontainers/runtime-spec/blob/main/features.md) of
+              the OCI Runtime Spec, in a JSON string representation.
+            type: "string"
```

Comment:
There was some discussion around this part; I'll open a proposal soon (should be a minor change)